### DISCUSSION TITLE
Any help to automatically copy image file name to the ALT tag?

### DISCUSSION THREAD

**Discussion by Vinc26** [GetPublii/Publii#1808] [Help] [answered]
Hi there :)

It's as simple as that: I have lot of images, very well SEO named, like "Xxxxxx-xxxx-xxxxxx-xxxxx-xxxxx-xxx.jpg". And I just want that Publii converts automatically the file name to the ALT tag of the image.

Any help?

**Comment by Vinc26** [comment]
No idea? 😬

**Reply by michaelfluegge** [reply]
this was being asked several times. Everybody is hoping/waiting for a solution...

**Comment by Vinc26** [comment]
Perhaps, a simple plugin could do that? Sad I'm only very very beginner developper to try that.

**Comment by gpsblues** [comment, marked as answer]
Hi everyone,

I found the challenge both interesting and useful for saving a lot of time when file names already have SEO value.

I’ve released a pre-release plugin version for testing.
https://github.com/gpsblues/Publii-Plugin-Alt-Filler

Let me know what you think! :)
🍻

**Reply by Vinc26** [reply]
Thank you so much!

**Comment by michaelfluegge** [comment]
- Alt Text is a brief text description of an image.
- Alt Text is read by screen readers (or other assistive tech) to provide information about the image's purpose to the reader.
- Alt text should not just describe what the image contains, but instead should provide context on how the image relates to the page content.

https://accessibility.huit.harvard.edu/describe-content-images

So ALT is more than a word or the name of the image. But that is my personal view on that topic, only.

**Comment by Vinc26** [comment]
Yes, that's how I create the file names of my images too.
